Userland/Applications/PixelPaint/EllipseTool.cpp

```diff
@@ -78,6 +78,10 @@ void EllipseTool::on_mousemove(Layer*, MouseEvent& event)
         return;
 
     m_ellipse_end_position = event.layer_event().position();
+
+    if (event.layer_event().shift())
+        m_ellipse_end_position = m_ellipse_start_position.end_point_for_square_aspect_ratio(event.layer_event().position());
```

Comment:
Seems odd to write the value twice in the shift case, how about:

```suggestion
    if (event.layer_event().shift())
        m_ellipse_end_position = m_ellipse_start_position.end_point_for_square_aspect_ratio(event.layer_event().position());
    else
        m_ellipse_end_position = event.layer_event().position();
```